content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -10,6 +10,8 @@ owner: istio/wg-policies-and-telemetry-maintainers
 test: no
 ---
 
+{{< boilerplate experimental-feature-warning >}}
```

Comment:
Normally [Experimental] is added to the end of the title as well.

Reply:
Done.